zuul.d/nodesets.yaml

```diff
@@ -536,6 +535,80 @@
       - name: controller
         nodes:
           - ubuntu-bionic
+- nodeset:
+    name: iosxr-7.0.2-python27
+    nodes:
+      - name: ubuntu-bionic
+        label: ubuntu-bionic-1vcpu
+      - name: iosxr-7.0.2
+        label: iosxrv-7.0.2
+    groups:
+      - name: appliance
+        nodes:
+          - iosxr-7.0.2
+      - name: controller
+        nodes:
+          - ubuntu-bionic
+
+- nodeset:
+    name: iosxr-7.0.2-python35
+    nodes:
+      - name: ubuntu-xenial
+        label: ubuntu-xenial-1vcpu
+      - name: iosxr-7.0.2
+        label: iosxrv-7.0.2
+    groups:
+      - name: appliance
+        nodes:
+          - iosxr-7.0.2
+      - name: controller
+        nodes:
+          - ubuntu-xenial
+
+- nodeset:
+    name: iosxr-7.0.2-python36
+    nodes:
+      - name: centos-8
+        label: centos-8-1vcpu
+      - name: iosxr-7.0.2
+        label: iosxrv-7.0.2
+    groups:
+      - name: appliance
+        nodes:
+          - iosxr-7.0.2
+      - name: controller
+        nodes:
+          - centos-8
+
+- nodeset:
+    name: iosxr-7.0.2-python37
+    nodes:
+      - name: fedora-32
+        label: fedora-32-1vcpu
+      - name: iosxr-7.0.2
+        label: iosxrv-7.0.2
+    groups:
+      - name: appliance
+        nodes:
+          - iosxr-7.0.2
+      - name: controller
+        nodes:
+          - fedora-32
+
```

Comment:
this likely all can remove, we mostly want to test against devel right now.  Given we didn't target our old collections for this version of IOSXR